### ISSUE TITLE
The library detects users as Webhook after the last Discord update

### ISSUE THREAD

**Issue by hunter8288** [not-ilinked/Anarchy#3362]
After the last Discord update, many users have registered nicknames for themselves, and their discriminators have become zero. Now the library defines such users as Webhook, I think this needs to be fixed, but I don’t know how.

\REST\User\User\DiscordUser.cs
```        
public DiscordUserType Type
        {
            get
            {
                if (Discriminator == 0)
                    return DiscordUserType.Webhook;
                else
                    return _bot ? DiscordUserType.Bot : DiscordUserType.User;
            }
        }
```

**Comment by purpl3r0se** [comment]
will be fixed in a few years.

**Comment by TheVisual** [comment]
https://github.com/not-ilinked/Anarchy/pull/3365

We need him to pull it. else just build from my source for now :)

**Comment by purpl3r0se** [comment]
fixed within #3366
